PowerPointLabs/PowerPointLabs/DataSources/ColorDataSource.cs

```diff
@@ -316,6 +481,69 @@ public HSLColor ThemeColorThirteen
             }
         }
 
+        #endregion
+
+        #region API
+
+        public void AddColorToRecentColors(HSLColor color)
+        {
+            List<HSLColor> recentColors = GetListOfRecentColors();
+
+            int index;
+            for (index = 0; index < recentColors.Count; index++)

```

Comment:
this for loop can be changed to 
`
int index = recentColors.IndexOf(color);
if (index == -1) {
index = recentColors.Count;
}
`

Reply:
Yes, I forgot about the existence of `IndexOf`. :P Thanks! updated PR